Networking/NetworkingTests/Mapper/SystemPluginMapperTests.swift

```diff
@@ -1,9 +1,9 @@
 import XCTest
 @testable import Networking
 
-/// SystemStatusMapperTests Unit Tests
+/// SystemPluginMapper Unit Tests
 ///
-class SystemStatusMapperTests: XCTestCase {
+final class SystemPluginMapperTests: XCTestCase {
```

Comment:
nit: some unit tests for `SystemStatusMapper` would be helpful, maybe in a separate PR!

Reply:
Yes, that's my intention. Thank you 😄